src/index.js

```diff
@@ -264,7 +264,7 @@ export default class GoCommerce {
 
   payment(paymentDetails) {
     const {order_id, amount, provider, stripe_token, paypal_payment_id, paypal_user_id} = paymentDetails;
-    if (order_id && amount && provider && (stripe_token || (paypal_payment_id && paypal_user_id))) {
+    if (order_id && amount !== undefined && provider && (stripe_token || (paypal_payment_id && paypal_user_id))) {
```

Comment:
lets change this to `amount != null` so we cancel on both `null` and `undefined` but not on `0`